src/include/kdbopmphm.h

```diff
@@ -14,30 +14,43 @@
 /**
  * For usage look in [datastructures.md](/doc/dev/datastructures.md)
  *
+ * Theoretical limit of elements:
+ *
+ * The whole OPMPHM is limited to the opmphmHashfunction (...) that returns a uint32_t.
```

Comment:
Just out of curiosity: Which effort would it be to change to 64bit, and which performance impact would it have? (Maybe describe it in your future work.)